interface/src/app/gene/resource.js

```diff
@@ -0,0 +1,22 @@
+angular.module( 'adage.gene.resource', ['ngResource'])
+
+.factory( 'Gene', ['$resource', function($resource) {
+  return $resource(
+    '/api/v0/gene/:id', {id: '@id'}, {
+
+      query: {
```

Comment:
For the most part - I guess the only difference is that isArray: is usually set to 'true'. However, I don't think we are using 'query' for now - I was including it in there in case we used it at some point, but I guess we can add it to the code if we need it, right?